requirements/docs.txt

```diff
@@ -1,6 +1,6 @@
 sphinx ==5.3.0
 myst-parser ==1.0.0
-nbsphinx ==0.9.4
+nbsphinx ==0.9.5
```

Comment:
```suggestion
nbsphinx ~=0.9.5
```

Reply:
in this case, shall it be `nbsphinx ~=0.9`?